src/test/java/com/google/api/codegen/testdata/go_main_library.baseline

```diff
@@ -686,6 +670,13 @@ type BookIterator struct {
     items    []*librarypb.Book
     pageInfo *iterator.PageInfo
     nextFunc func() error
+
+    // internalFetch is for use by the Google Cloud Libraries only.
+    //
+    // internalFetch returns results from a single call to the underlying RPC.
+    // The number of results is no greater than pageSize.
+    // If there are no more results, nextPageToken is empty and err is nil.
+    internalFetch func(pageSize int, pageToken string) (results []*librarypb.Book, nextPageToken string, err error)
```

Comment:
I made the change to export it on a config but forgot to change the config. Done.